suprb/base.py

```diff
@@ -51,7 +51,7 @@ def _more_str_attributes(self) -> dict:
         return {}
 
 
-class BaseRegressor(BaseEstimator, RegressorMixin, metaclass=ABCMeta):
```

Comment:
we do we need this ordering change?

Reply:
Needs to be in this order for the new sklearn version: https://scikit-learn.org/stable/modules/generated/sklearn.base.RegressorMixin.html